Poker.FE/Assets/MainMenu/Textures/loading_animation.anim

```diff
@@ -0,0 +1,193 @@
+%YAML 1.1
```

Comment:
semi write it, i used a tool for know what's im doing here. this part is the progress bar animation between scene and the initial loading of game resources. took alot of time to implement 😫